crates/client/Cargo.toml

```
@@ -10,8 +10,10 @@ homepage = { workspace = true }
 repository = { workspace = true}
 
 [features]
-default = []
+default = ["cli-interactive"]
 native-tls-vendored = ["reqwest/native-tls-vendored"]
+cli-interactive = ["dialoguer"]
```

nit; not much functional difference but prevents an implicit feature named "dialoguer" from being created

```suggestion
cli-interactive = ["dep:dialoguer"]
```

---

Fixed